app/api/auth.py

```diff
@@ -56,7 +55,7 @@ def login():
       })
    if hr is not None and "integrate_with_hr" in hr:
 
-       URL_login = 'https://hr.excellencetechnologies.in/attendance/API_HR/api.php'
+       URL_login = 'http://dev.hr.excellencetechnologies.in/hr/attendance/API_HR/api.php'

```

Comment:
this should come from config

Reply:
ok this one we can access from config.py